include/faker-cxx/Person.h

```diff
@@ -105,7 +105,7 @@ class Person
      * Person::sex() // "Male"
      * @endcode
      */
-    static std::string sex();
+    static std::string sex(Language language = Language::English);
```

Comment:
please add additional tests, it would be good to have tests automatically for each language like for names, check PersonTest test with param:

class PersonTest : public TestWithParam<Country>

...

INSTANTIATE_TEST_SUITE_P(TestPersonNamesByCountries, PersonTest, ValuesIn(countries),
                         [](const TestParamInfo<Country>& info) { return generatedTestName.at(info.param); });
                         
You could add new test suite like PersonSexSuite.

Reply:
added few test cases, let me if that's what was expected